src/CSModule.sol

```diff
@@ -125,6 +130,9 @@ contract CSModule is
         _pauseFor(PausableUntil.PAUSE_INFINITELY);
     }
 
+    /// @dev should be called after update on the proxy
+    function finalizeUpgradeV2() external reinitializer(2) {}
```

Comment:
Do we still need this method? It is now empty.

Reply:
I think it is crucial to update contract version somehow